src/binary-encoding.adoc

```diff
@@ -851,6 +851,8 @@ The following letters to are used to denote bits in each subfield in diagrams th
 order in all encoding variants, and consume the same number of bits, with the exception of the fractional bits, which
 consume only enough bits to represent the fractional precision supported by the opcode being used.
 
+The `Month` and `Day` subfields are one-based—`0` is not a valid month or day.
```

Comment:
The punctuation here is a bit tricky; I read `one-based-0` as a single term. Consider using a semicolon or breaking it into its own sentence.
```suggestion
The `Month` and `Day` subfields are one-based; `0` is not a valid month or day.
```